origin.spec

```diff
@@ -567,22 +574,10 @@ fi
 %{_prefix}/lib/tuned/%{name}-node-host
 %{_prefix}/lib/tuned/%{name}-node-guest
 %{_mandir}/man7/tuned-profiles-%{name}-node.7*
-
-%post -n tuned-profiles-%{name}-node
-recommended=`/usr/sbin/tuned-adm recommend`
-if [[ "${recommended}" =~ guest ]] ; then
-  /usr/sbin/tuned-adm profile %{name}-node-guest > /dev/null 2>&1
-else
-  /usr/sbin/tuned-adm profile %{name}-node-host > /dev/null 2>&1
-fi
-
-%preun -n tuned-profiles-%{name}-node
-# reset the tuned profile to the recommended profile
-# $1 = 0 when we're being removed > 0 during upgrades
-if [ "$1" = 0 ]; then
-  recommended=`/usr/sbin/tuned-adm recommend`
-  /usr/sbin/tuned-adm profile $recommended > /dev/null 2>&1
-fi
+%{_prefix}/lib/tuned/openshift
+%{_prefix}/lib/tuned/openshift-control-plane
+%{_prefix}/lib/tuned/openshift-node
+%{_sysconfdir}/tuned/recommend.conf
```

Comment:
The profile changes automatically to the recommended profile once the "tuned" daemon is restarted by the ansible installer (see https://github.com/openshift/openshift-ansible/pull/4566).

Yes, I was thinking of using something like tuned-profiles-openshift instead of tuned-profiles-%{name}-node for both an openshift master and nodes.  Would that be OK?  Not sure all masters should be nodes.  Are you sure about that?  I've installed OpenShift with a master without being a "node", meaning, I'd have to adjust https://github.com/openshift/openshift-ansible/pull/4566 too.  That's why I've created an ansible role for tuned, to be included not only from nodes, but masters too eventually.